cmd/plan.go

```diff
@@ -170,10 +170,10 @@ func GetPlanCommand() *cobra.Command {
 		Run:   func(cmd *cobra.Command, _ []string) { planHandler(cmd, flags) },
 	}
 
-	planCmd.Flags().StringVarP(&flags.srcpath, sourceFlag, "s", "", "Specify source directory.")
+	planCmd.Flags().StringVarP(&flags.srcpath, sourceFlag, "s", "", "Specify source directory or a VCS url.")
```

Comment:
Can you change VCS to `git`? Can you make this change in all the places?

Once you are ready with the git-support page, you can reference the link here in a `(` and `)`